### DISCUSSION TITLE
Document Jamulus related projects on the website (in KB)

### DISCUSSION THREAD

**Discussion by ann0see** [jamulussoftware/jamulus#2270] [Documentation]
Currently we have https://github.com/jamulussoftware/jamulus/blob/master/RELATED-PROJECTS.md
… a file linking to Jamulus related projects. Should we extend and move the file to the community KB? What do you think?

I‘d like to have some tools from @dtinth in there too.

**Comment by gilgongo** [comment]
Makes sense to have them on the website KB I think, as many people won't think to look in the repo.

**Comment by ann0see** [comment]
Ok: https://github.com/jamulussoftware/jamuluswebsite/commit/652b244921eaef7dee1a7eca9d7af6b852fde94a

documented it

**Comment by ann0see** [comment]
@softins I've now copied your related projects file to the website.

https://jamulus.io/de/kb/2022/01/25/Related-Projects.html
